### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by jesscxu**
initialized the node package, set up configurations, and set up the server so that the application can now run

### CODE REVIEW

File: config/development.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  PORT: 3000
+};
+
+
+
```
Comment (mgartner) [inline config/development.js]: nit: remove extra 3 extra lines at the end of the file

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const Movies = require('./server');
+
+Movies.start(() => {
+  Util.log(`Server started on port: ${Config.PORT}`);
+});
```
Comment (mgartner) [inline lib/index.js]: nit: add a new line here. let's talk about this one IRL

File: lib/server.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+const server = new Hapi.Server({
+  connections: {
+    router: { stripTrailingSlash: true } // removes trailing slashes on incoming paths
+  }
+});
+
+server.connection({ port: Config.PORT });
+
+module.exports = server;
```
Comment (mgartner) [inline lib/server.js]: nit: add a new line here

**Review by mgartner — CHANGES_REQUESTED** [review]

**Review by mgartner — APPROVED** [review]
LGTM

File: lib/index.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Util = require('util');
```
Comment (itsachen) [inline lib/index.js:3]: Nit: Generally we separate external and internal file imports with a newline, as well as alphabetize them and align their `=`'s. See [this section](https://github.com/lob/lob-api/blob/master/src/plugins/features/addresses/index.js#L3-L10) from `lob-api` as an example.

**Review by itsachen — APPROVED** [review]
LGTM!

**Pull request merged**